app/components/emoji/emoji.js

```diff
@@ -49,8 +50,13 @@ export default class Emoji extends React.PureComponent {
             return <Text style={textStyle}>{literal}</Text>;
         }
 
+        let ImageComponent = WebImage;
+        if (Platform.OS === 'android') {
+            ImageComponent = Image;
```

Comment:
@jarredwitt If iOS isn't caching these, could it be not caching profile pictures or file attachment previews? I'm wondering if it would make sense to use this there as well